library.rb

```diff
@@ -0,0 +1,61 @@
+require 'date'
+
+require 'json'
+
+# Load the data from the files
+
+# Load the data from the files
+
+books_data = File.read('books.json')
+
+people_data = File.read('people.json')
+
+rentals_data = File.read('rentals.json')
+
+# Parse the JSON data into Ruby hashes
+
+books = JSON.parse(books_data, symbolize_names: true)
+
+people = JSON.parse(people_data, symbolize_names: true)
+
+rentals = JSON.parse(rentals_data, symbolize_names: true)
+
+# Display the data in the terminal
+
+puts 'Books:'
+
+books.each do |book|
+  puts "Title: #{book[:title]}"
+
+  puts "Author: #{book[:author]}"
+
+  puts "Rentals: #{book[:rentals].size}"
+
+  puts
+end
+
+puts 'People:'
+
+people.each do |person|
+  puts "Name: #{person[:name]}"
+
+  puts "Age: #{person[:age]}"
+
+  puts "Parent Permission: #{person[:parent_permission]}"
+
+  puts "Rentals: #{person[:rentals].size}"
+
+  puts
+end
```

Comment:
- [ ] When I run the app and add some books and students and teachers and exit and re-run the app it doesn't preserve data as expected it should display all the books and people that were created previously so please make sure to fix that in order to complete the requirements 
- and you should also handle the case when a `(file).json` is missing or empty it should create a file then